inst

```diff
@@ -67,7 +69,12 @@ nextcloud_main() {
     fi
     nextcloud_ensure_ucr
     nextcloud_attempt_memberof_support
-    nextcloud_configure_saml "$@"
+    if is_ucr_true appcenter/apps/nextcloud/oidc-login; then
+        univention-app shell nextcloud sudo -u www-data php /var/www/html/occ app:disable user_saml || true
+        nextcloud_configure_oidc "$@"
+    else
+        nextcloud_configure_saml "$@"
```

Comment:
i suppose it makes sense to disable user_oidc in this case as well, analogue to first if branch

Reply:
I re-checked and saw that those apps don't conflict with each other. If SAML is enabled it shows a login dialog saysing wheather to login via "direct login" or "SAML / SSO login". When clicking on "direct login" there is a integrated button for the login via OIDC.

Nice would be, if the SAML login plugin wouldn't replace the login dialog but instead just add the same button as OIDC does to the original login page.

So, my suggestion now is to not disable any of these apps. Or make it explicitly configurable via UCR, which app should be disabled.

Reply:
I adjusted the PR accordingly, see the updated MR description.

Reply:
> Nice would be, if the SAML login plugin wouldn't replace the login dialog but instead just add the same button as OIDC does to the original login page. 

Mh, SAML is configured to allow using the other auth mechanisms.